CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 ## Changelog
 
+### 0.13.0
+
+**Commit Delta**: [Change from 0.13.0 release](https://github.com/plus3it/watchmaker/compare/0.12.1...0.13.0)
+
+**Released**: 2018.01.29
+
+**Summary**:
+
+*   add new salt formula amazon-inspector-formula
```

Comment:
Can go ahead and summarize the change to the class structure, including a link to the PR. See the Changelog entry for [0.9.5](https://github.com/plus3it/watchmaker/blob/develop/CHANGELOG.md#095) as an example.

Also, here's the format we used last time we added a formula, might as well keep it consistent:

```
*   nessus-agent-formula
    -   New salt formula distributed with Watchmaker
```

Reply:
Done